.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -9,6 +9,9 @@ assignees: ''
 
 ## Bug Report
 
+<!-- Check this if you would like to implement a PR, we are more than happy to help you go through the process !-->
+- [ ] I want to implement a PR
```

Comment:
Howabout something more personal like, "I would like to work on this!" or "I would like to create a PR for this!"